iothub/device/src/Convention/WritableClientProperty.cs

```diff
@@ -32,10 +32,7 @@ internal WritableClientProperty(long version, PayloadConvention payloadConventio
         /// </summary>
         public string PropertyName { get; internal set; }
 
-        /// <summary>
-        /// The value of the property for which an update request is received.
-        /// </summary>
-        public object Value { get; internal set; }
```

Comment:
Changed this to internal so that users are encouraged to discover "TryGetValue".